tests/manage/z_cluster/test_drain_node_mon.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import pytest
+
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check
+from ocs_ci.ocs.resources.pod import get_mon_pods
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes, get_node_objs
+from ocs_ci.helpers.helpers import get_mon_pdb
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_external_mode
+@skipif_ocs_version("<4.6")
+@bugzilla("1959983")
+class TestDrainNodeMon(ManageTest):
+    """
+    1.Get worker node name where monitoring pod run
+    2.Verify pdb status, disruptions_allowed=1, max_unavailable_mon=1
+    3.Drain node where monitoring pod run
+    4.Verify pdb status, disruptions_allowed=0, max_unavailable_mon=1
+    5.Verify the number of mon pods is 3 for (1400 seconds)
+    6.Change node to be scheduled
```

Comment:
I think the step to do operator restart is missing, please check

Reply:
@Shrivaibavi Do I need to restart operator pod?

Reply:
yes oded, please see the comment for more details https://bugzilla.redhat.com/show_bug.cgi?id=1959983#c8

Reply:
done

Reply:
done